batch2/batch/batch.py

```diff
@@ -74,16 +74,16 @@ def _copy_container(name, files):
                 mount_path='/gsa-key',
                 name='gsa-key')]  # FIXME: this shouldn't be mounted to every container
 
-        if self._pvc_name is not None:
+        if self.pvc_size or (self.input_files or self.output_files):
```

Comment:
self.pvc_size is not used anywhere

Reply:
Also, where is the default PVC size used if there are input files or output files but no pvc size listed.

Reply:
Nowhere at the moment, again, because it isn't used.

The next step is to send the job_spec directly to the worker and rip out all the legacy k8s stuff.  The worker can decide what to do.

Reply:
ok.